crates/red_knot_python_semantic/resources/mdtest/import/relative.md

```diff
@@ -219,23 +219,11 @@ import package
 reveal_type(package.foo.X)  # revealed: Unknown
 ```
 
-## In the src-root
+## Relative imports at the top of a search path
 
-`parser.py`:
-
-```py
-X: int = 42
-```
-
-`__main__.py`:
-
-```py
-from .parser import X
-
-reveal_type(X)  # revealed: int
-```
-
-## Beyond the src-root
+Relative imports at the top of a search path result in a runtime error:
+`ImportError: attempted relative import with no known parent package`. That's why Red Knot should
```

Comment:
It also sneaks in a prose usage of "Red Knot", in the eternal "Red Knot" vs "red-knot" wars ;)

Reply:
cc. @BurntSushi